static/stories/publication.md

```diff
@@ -9,15 +9,77 @@ thumbnail: "nlesc-dummy.png"
 
 :::Chapter{headline="Intro slides" presentation="slides-publication.md"}
 
-:::Chapter{headline="Publishing resources"}
+:::
+
+:::Chapter{headline="Software publication pipeline" widemd=1}
+
+## Steps
+It is recommended that all steps in the pipeline are followed, but they are not dependent on eachother. In this pipeline we provide resources to achieve the whole pipeline using GitHub or GitLab for hosting the source code 
+* Software repository
+* Tag/semantic versioning
+* Get a persistent identifier
+* Registries
+* Software Paper
+
 
 ## Software repository
-Popular repository hosting at GitHub
-## Tag/semantic versioning
-See a description of the semantic versioning scheme here: <https://semver.org/>
-## Persistent Identifier
+If possible, during the development of Research Software, changes should be tracked using dedicated version control software like [git](https://git-scm.com/).  
+Put all source files in a public version-controlled repository, preferably GitHub or GitLab. Both GitHub and Gitlab provide online repositories for hosting source code. The nice thing about GitHub and GitLab is that it also stores the history of changes to the source code. As can be inferred from their names, they were designed to be used together with Git version control.  
```

Comment:
```suggestion
Put all source files in a public version-controlled repository, preferably GitHub or GitLab. Both GitHub and Gitlab provide online repositories for hosting source code. As can be inferred from their names, they were designed to be used together with Git version control.  The nice thing about using git together with GitHub and GitLab is that it stores the complete history of changes to the source code, rather than just a single version . 
```